jwst/datamodels/schemas/pixelarea.schema.yaml

```diff
@@ -1,5 +1,13 @@
 allOf:
 - $ref: referencefile.schema.yaml
+- $ref: nirspec_area.schema.yaml
```

Comment:
nirspec_area.schema.yaml is the schema for a completely separate flavor of pixel area reference file that is unique to NIRSpec spectroscopic modes and hence does not belong here amongst the pixel area ref file for imaging modes.